go/ql/lib/semmle/go/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -188,6 +188,12 @@ predicate clearsContent(Node n, Content c) {
   // FlowSummaryImpl::Private::Steps::summaryClearsContent(n, c)
 }
 
+/**
+ * Holds if the value that is being tracked is expected to be stored inside content `c`
+ * at node `n`.
+ */
+predicate expectsContent(Node n, ContentSet c) { none() }
```

Comment:
```suggestion
predicate expectsContent(Node n, ContentSet c) {
  FlowSummaryImpl::Private::Steps::summaryExpectsContent(n, c)
}
```
This would match Java

Reply:
👍🏻 I was going from the original PR, but it's obviously been changed since then.